talk/morelanguage/stl.tex

```diff
@@ -58,7 +58,7 @@
     \item examples
       \begin{itemize}
       \item string
-      \item tuple, list, vector, deque
+      \item tuple, list, forward\_list, vector, deque
```

Comment:
Forward list is C++11, should we change the slide header? I wanted to include it here though, since we give an overview.

Reply:
No opinion on that. It's good for me with and without.

Reply:
I added C++11 in parenthesis. I also noticed `std::array` is missing. And I replaced `hash_` with `unordered_`.